lib/ftp.js

```diff
@@ -200,7 +200,25 @@ class Client {
         // The directory is part of the return message, for example: 
         // 257 "/this/that" is current directory.
         return res.message.match(/"(.+)"/)[1];
-    } 
+    }
+
+    /**
+     * Get the last modified time of a file.
+     * 
+     * @param {string} filename  Name of the file in the current working directory.
+     * @returns {Promise<Date>}
+     */
+    async lastMod(filename) {
+        const res = await this.send("MDTM" + filename);
```

Comment:
This won't work, missing whitespace after `MDTM`

Reply:
oh no